tests/integration/test_integration_app.py

```diff
@@ -0,0 +1,49 @@
+import trello_items as trello
+import pytest
+import requests
+from app import create_app
+from dotenv import load_dotenv, find_dotenv
+from card import Card
+import json 
+
+class MockResponse:
+    def __init__(self, json_data, status_code):
+        self.json_data = json_data
+        self.status_code = status_code
+
+    def json(self):
+        return self.json_data
+
+@pytest.fixture
+def client():
+    # Use our test integration config instead of the 'real' version 
+    file_path = find_dotenv('/DevOps-Course-Starter/tests/.env.test')
```

Comment:
Specifying the full path is usually a bit of a code smell - this will only work correctly if it is in the perfect folder structure on every machine. Notably this means that running tests locally (not in docker) for me failed. Your previous approach of just specifying the appropriate filename `.env.test` should work both in and out of docker though